db/fixtures/miq_product_features.yml

```diff
@@ -864,6 +864,14 @@
       :description: Edit Tags of Backups
       :feature_type: control
       :identifier: cloud_volume_backup_tag
+    - :name: Restore Backup
+      :description: Restore Backup to Volume
+      :feature_type: control
+      :identifier: cloud_volume_backup_restore_to_volume
```

Comment:
How about `cloud_volume_backup_restore`?

Reply:
@agrare identifiers should be uniq and `cloud_volume_backup_restore` is already there

Reply:
Ah okay :+1:

Reply:
Hm is that feature used anywhere?  It sounds like exactly what this does just defined on `CloudVolume` but I don't see any restore methods on `CloudVolume`

Reply:
@agrare if you mean `cloud_volume_backup_restore_to_volume`, then it`s used here https://github.com/ManageIQ/manageiq-ui-classic/pull/2037

Reply:
Probably, I need a better name for this feature.

Reply:
@alexander-demichev actually I was wondering if `cloud_volume_backup_restore` could be removed from `CloudVolume` and moved here?

Reply:
@agrare It`s used [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/d3f5ad63ca0fe4c4f0d1c828e4bc826ea27d322a/app/controllers/cloud_volume_controller.rb#L491)

Reply:
Alright maybe we can refactor this later to use this instead?